Tests/LibWeb/Text/input/set-timeout-with-no-active-script.html

```diff
@@ -0,0 +1,9 @@
+<script src="include.js"></script>
+<script>
+</script>
+<script>
+    test(setTimeout.bind(this, () => {}));
+    test(() => {
+        println("didn't crash!");
+    })
```

Comment:
could you make this test call `test()` only once. that is implicit rule my work in https://github.com/SerenityOS/serenity/pull/21073 relies on :) actually we should probably make test fail if `test()` is called more than once within single page.